en/developer_guide/routing.rst

```diff
@@ -0,0 +1,79 @@
+Routing
+=======
+
+Base URL
+--------
+
+The page base url is compose with domain name and language prefix.
+To manage all possibilities, Open Orchestra associate each website in database with aliases that contain:
+
+* **domain**: domain name
+* **language**: language prefix
+* **scheme** : to choose with which protocol page can be access
+
+Here is an example of some base URL generated for a website associate to two domains ``dev.example.com`` and ``test.example.com`` with english and french languages and default languages english.
+
++----------+--------+------------------+----------+-----------------------------+
+| Alias Id | Scheme | Domain           | Language | Base URL                    |
++==========+========+==================+==========+=============================+
+| 0        | http   | dev.example.com  | fr       | http://dev.example.com      |
++----------+--------+------------------+----------+-----------------------------+
+| 1        | http   | dev.example.com  | en       | http://dev.example.com/en   |
++----------+--------+------------------+----------+-----------------------------+
+| 2        | http   | test.example.com | fr       | http://test.example.com     |
++----------+--------+------------------+----------+-----------------------------+
+| 3        | https  | test.example.com | en       | https://test.example.com/en |
++----------+--------+------------------+----------+-----------------------------+
+
+
+URL Pattern
+-----------
+
+The url pattern attribute is generated automatically with the name of the page or define by user for redirection.
+It represents the pattern that will be the URL of the page or redirection on the website.
+
+See ``Url pattern`` chapter from `Node Configuration`_ for more details.
+
+Implementation
+--------------
+
+``open-orchestra-front-bundle`` contain all routing files in the ``routing`` folder.
+
+
+Loader
+~~~~~~
+
+On Open Orchestra, ``Loader`` provide route from database collections:
+
+* ``DatabaseRouteLoader`` generate routes for different nodes.
+Routes in loader are link with ``NodeController`` and take languages, node, site and alias in parameters.
```

Comment:
Routes generated are linked to ``NodeController`` and contains the alias language, the node id, the site id and the alias id